RoddyCore/src/de/dkfz/roddy/config/ToolEntry.java

```diff
@@ -217,6 +219,39 @@ public ToolStringParameter clone() {
         }
     }
 
+    public static class ToolFileParameterCondition {
```

Comment:
Maybe the name is wrong.  It is the condition for check. If you do not want to set true/false, you can set a conditional variable instead.